tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/ExternalSortExec.java

```diff
@@ -630,19 +636,43 @@ public TableStats getInputStats() {
     CLOSED
   }
 
+  private static class SimpleMerger extends PairWiseMerger {
```

Comment:
I'm thinking of rewriting current up-heap to down-heap to reduce comparing further (50% for average AFAIK). So I've named it randomly. Till then let's name it VectorComparePairWiseMerger.